WordPress/Classes/Extensions/String+Helpers.swift

```diff
@@ -30,4 +30,13 @@ extension String {
         }
         return self
     }
+
+    /// Returns a string without the character at the specified index.
+    /// This is a non-mutating version of `String.removeAtIndex(_:)`.
+    func removingCharacterAtIndex(index: Index) -> String {
```

Comment:
Maybe `removeCharacterAtIndex` vs `removingChar...`  The gerund tense just sounds weird to me.